### ISSUE TITLE
AGP 2: Aragon Community Governance Model

### ISSUE THREAD

**Issue by luisivan** [aragon/governance#3]
### Goals

We want a productive, happy and agile community that can welcome and bring forth new ideas.

We will rely on 5 key principles:

- Independence
  - Decision-making structure
  - Independence from employment relations
- Pluralism
  - Diversity of contributors
  - Management of conflict
  - Determination of leadership—processes and turnover
- Representation
  - Membership structure and rights
  - Ways in which those in authority are accountable to contributing members
  - Degree to which members can exercise voice on community members
- Decentralized decision-making
  - How contributors gain code commit and #frontline rights
  - How contributors gain access to the project’s decision-making structure
  - The degree to which project communications and activities are publicly available
- Autonomous participation
  - Newcomer joining rate
  - Newcomer socialization opportunities
  - Rate of idea generation

### Description

Our Code of Conduct and Community Guidelines act as a basis for all our community activity, how to participate, how to grow our community, how to represent Aragon to others outside the community.

Proposed meetings:

- **Dev meeting**: Bi-weekly, Fridays on uneven weeks. Time 14.00 CEST.
  - Discussion on current technical development, what has got done, what needs to get done for the next release etc.
  - Facilitator: Tech Lead 
  - Participants: Aragon Dev team + Any Aragon team members who want to participate + possible relevant contributors
  - Viewers: Public on Youtube

- **Aragon HQ**: Monthly, Last Friday of the month. Time 15.00 CEST.
  - Discussion about the state of Aragon, near future plans and decisions that need to be made. 10 min presentations with slides from team members, in which each one will run through the most important things we have done during the month, the problems they have found, and what they plan to do next month (that can be a bit blurry, and that's fine)
  - Facilitator: Project Lead
  - Participants: All Aragon team members + possible relevant contributors
  - Viewers: Public on Youtube

- **Team hangout**: Weekly, Friday. Time 15.00 CEST
  - Relaxed weekly meeting with the team where anyone can talk about things they feel are current
  - Facilitator: If needed, any of the participants
  - Participants: Project Lead + anyone from the Aragon team who wants to participate
  - Viewers: Non-public

- **Community meeting**: Monthly, second Friday of the month. Time 16.00 CEST.
  - Discussion about Aragon, questions from the community members answered by the team. AMA will be conducted in [/r/aragonproject](https://www.reddit.com/r/aragonproject/)
  - Facilitator: Project Lead/Community Lead
  - Participants: Project Lead + Community Lead + Aragon team members who want to participate
  - Viewers: Public on Youtube

Structures:
- Project Lead or Tech Lead will have final call on all things pertaining to their area of expertise.
- Decisions made after thorough conversation on Meetings, informal vote to be taken if necessary, possible conflicts discussed if necessary, final vote if needed and decision confirmed by Project/Tech Lead.
- Facilitators are responsible for gathering the agenda for the Meetings where applicable, as well as either taking notes of the Meeting and decisions made or delegating this responsibility to another participant
- Use (The Etherian’s or our own) Colony to have community members participate and get rewarded for their non-code contributions
- Implement a system similar to [Maker’s Actions](http://makerdao.com/actions/) and post a monthly transparency report about the use of funds to the blog
- Institute Aragon Improvement Proposal, AIP, system for community members to create proposals about Aragon. Take a similar approach for governance with Aragon Governance Proposals (like this one). GitHub Issues will be used as a forum to discuss proposals by community and team members
- Proposals and ideas from the team and community members will be go through the AIP/AGP process and will then be discussed in the relevant Meetings
- Community Meetings will be AMA styled video meetings where team members will answer questions by the community members. Questions will be submitted through Reddit and answered live
- We’ll have templates that can/will be used for the basis of agendas in Meetings so the Facilitator can easily fill out each Meetings info, like who will be presenting what, who has slides etc.
- Facilitator of the Meeting will make sure that the discussion stays on rails, everyone gets a chance to speak their mind, everything on the agenda is discussed, necessary decisions are made and that the Meeting length stays on a reasonable length

### Uncertainties

Any right now, but feedback is very welcome! And thanks @kennyrowe for the feedback on this and @Smokyish for the initial draft and putting everything together!

**Comment by AaronFoster** [comment]
This is fantastic. The YouTube stuff will be great for acquiring new community members

**Comment by fadda** [comment]
Fantastic!
Such approach should become standard for every publicly funded project on the blockchain.

**Comment by izqui** [comment]
I agree with everything, good job Luis.

Let me add my two wei: 
- I think it would be pretty cool to start doing voting on the blockchain (even if a really simplified version). Love [what d0x will do](https://blog.district0x.io/district0x-dev-update-june-27th-2017-780bc20a04dc
) with their carbon voting thing. We have 4k+ holders who would love to feel heard, voting could done in a completely different contract so people's tokens are never in danger (we could even do it offchain with signatures.)
- We should create a new repo for the meeting calendar, notes and important links can be easily found.

**Comment by Smokyish** [comment]
@izqui The idea would be to utilize the upcoming wiki for the notes, links etc.

**Comment by AaronFoster** [comment]
@izqui that would be excellent (2 Wei, Lol)

When do you propose to start the community meetings/team hangouts @luisivan

**Comment by luisivan** [comment]
Agree with both using Carbon Vote and creating a new repo for important resources, or as @Smokyish said, using the wiki.

@AaronFoster I propose to start ASAP! So we can do the first ones during July

**Comment by Smokyish** [comment]
We are looking to finalize this soon, so please leave your feedback regarding this model during this week.

**Comment by AaronFoster** [comment]
@luisivan I can't wait. @Smokyish I haven't got any other than bring on July. You guys have clearly thought this through well.

My only concern is getting non technical people onto github.

**Comment by schmidsi** [comment]
This looks really inspiring. Good work. Maybe the [Sociocracy 3.0 framework](http://sociocracy30.org/) could also be considered.

**Comment by miguelprados** [comment]
Sorry for the long post, regarding what @luisivan and @izqui commented, I will reframe the discussion with a real example:

I plan to register my startup’s comunitaria.com SHA and initial capital distribution among partners in AN, so I will invest the equivalent euros into ETH with the agreed percentage distribution with my partners.

Then I have to legally register my company in Spain. For the Bylaws to be approved I need to submit a bank certificate with the initial bank deposit certificate of the declared shared capital (common stock), instead, I will submit the smart contract that AN can provide that we’ve effectively invested that capital (equivalent) in ETH. Most probably the notary will deny the registration, then I can initiate a legal claim for that smart contract to be declared valid, otherwise, if one of my partners starts a claim, I will be defenceless.

That legal claim, if successful, will open the doors to any company in Spain to legally register their companies within the AN. As I see it, whether to support or follow (or not) from Aragon that legal claim, can’t be a decision taken only by Aragon’s team and/or shareholders of ANT, it has to be a decision taking also globally with all the companies registered in AN or those registered or planning to be registered in Spain.

For Aragon, as a disintermediation company, getting rid of the notaries could be the goal to follow, but on the other hand, many companies unable to legally register their companies can be discouraged to use AN at all (when the first legal claim is pursued, maybe lost, and make it public). It’s not an easy decision to take, as it is not easy to change the world, but we should try.

As planned in AGP 2: Aragon Community Governance Model #3: Dev meeting, Aragon HQTeam and Team hangout are closed spaces for taking decisions by the Aragon’s team plus some selected contributors. The Community meeting are just a Q/A forum. Institute Aragon Improvement Proposal, AIP, are open proposal with higher community interactions but unrestricted to a particular stakeholder (for example companies with a registered SHA in Aragon).

The proposal to include a token for ANT holders to vote using carbon voting doesn’t solve the issue because, if asked, probably they will vote against any intention to enter any legal battle to legally register a company, because their interest is disintermediation and ANT value (tell me the forum, let me design the question, and I will tell you the outcome before it comes, and that’s apparent democracy, not real one). Also carbon voting is a traditional approach to voting, more shares more voting power. One person one vote is the correct approach in decentralised decisions, using a BITNATION WORLD CITIZENSHIP ID I think is a better approach and can lead to very interesting synergies between Aragon and Bitnation.

In summary, the question to be asked should drive the Who and How to vote taking the appropriate stakeholders and diversity (techies and non-techies :) onboard.

I am not a techie myself, but I’ve made (I think) the world’s first online open list candidates for local municipal elections in Spain back in 2011, curiously enough, Adan Sanchez de Pedro was the selected candidate for Malaga with only 18 years at that time. Since then I’ve been involved in taking online decisions for communities like the open city voting for the approval or denial of the current Seville’s mayor candidacy.

I am really glad to see that you are taking governance as a big thing as I think it is, I’d love to contribute if you find it useful.

**Comment by Smokyish** [comment]
@miguelprados This is the Community Governance Model and you're talking about Aragon Network Governance, which is quite different from this issue. The AN Governance Model has not been proposed yet, there are now two AGP's that would be parts of the larger sum that is AN Governance model, the AVT and Ranking system proposals. The Governance Model for AN is not something that will be rushed and pushed out in a hurry, it will take time to form the complete model along with feedback from our Advisors and community.

**Comment by miguelprados** [comment]
@Smokyish Community Governance and Aragon Network Governance will have areas of interaction as "districts" having "roads" in common. The example I made is just to ilustrate that the question to be asked is the driver of the Who and How to vote and the decisions may involve various stakeholders. As described, Dev meeting, Aragon HQTeam and Team hangout, are Aragon Team's internal meetings, and the Community meeting (as described) is just a Q/A forum. The only community governance is then the Institute Aragon Improvement Proposal, AIP. Maybe an idea is to start naming those Areas (districts) of decision to clarify.

**Comment by Smokyish** [comment]
@miguelprados I disagree.

Dev Meeting: 
- Anyone can create a new issue at https://github.com/aragon/aragon/issues
- Anyone can create a new issue at https://github.com/aragon/issues/issues
- Anyone can create a new AIP
- Anyone can join the #dev channel on our [Slack](https://aragon.chat) and ask Jorge, Luis, Harsh, me or anyone from the Dev Team to include an issue on the agenda.
- The meeting will be public and live on Youtube; There will be live chat and comments where anyone participate, bring up an issue or discuss things that are being discussed in the meeting.
- Anyone can request to join the meeting if they have made contributions to issues that will be discussed in the meeting or they just have made large contributions to the project, showing that they are interested and capable of helping the project.

Aragon HQ Meeting:
- Anyone can create a new issue at https://github.com/aragon/aragon/issues
- Anyone can create a new issue at https://github.com/aragon/issues/issues
- Anyone can create a new AIP or AGP
- Anyone can join the #general or #aragon-network channel on our [Slack](https://aragon.chat) and ask  Luis or me to include an issue on the agenda.
- The meeting will be public and live on Youtube; There will be live chat and comments where anyone participate, bring up an issue or discuss things that are being discussed in the meeting.
- Anyone can request to join the meeting if they have made contributions to issues that will be discussed in the meeting or they just have made large contributions to the project, showing that they are interested and capable of helping the project.

Community Meeting:
Yes, this is less formal and designed to be a way to engage and interact with the community, but does not exclude new issues being brought up, answered and discussed live. There will also be a live chat and comments on Youtube besides the questions from reddit, so anyone can participate in there as well.

The Meetings will take place at a time to cover many timezones, but not everyone might be able to join the live discussion, which is why there are multiple venues where issues can be brought up as well as discussed live, so saying that AIP's would be the only way to participate is just plain false.

**Comment by miguelprados** [comment]
Thanks, that clarifies some doubts. Adding issues (as in Dev Meeting and HQ meetings) is openness for proposals, not openness for decisions, which is perfectly fine, but naming is important and proposals are different from decisions, that's governance, and it is fine if the final decision on software development is closed to the team, there are no issues with that, as responsibles are the management team. That's why Areas (or districts) of decisions are a good way to construct a "constitution" for the Aragon community, which includes the Aragon Network (as a different area of discussion that overlaps). I am making these comments with the healthy intention to help, I hope the writing doesn't ruin that :)

**Comment by Smokyish** [comment]
We could make some classifications for proposals and decisions, categorize some as "community", ones where we take a community vote and others as "team" decisions for the community to have their voices be heard.

This will require some further planning on how to conduct the voting, where to publish the information about upcoming and on-going votes, how the get the information to as many people as possible, how to prevent sybil attacks, how to conclude that the community's voice has been heard, what conclusions to draw from the results of a vote, how and when to publicly present the data from the voting to the team and the community, etc. 

And i agree, there is bound to be overlap with the Community and AN Governance, we will keep both as flat and inclusive as possible, and there will a lot more of "community" issues/decisions with things related to the AN.

And this current system isn't trustless, it places trust in that the two founders have the best intentions of the project in mind.

**Comment by miguelprados** [comment]
@Smokyish That's a good approach. It's like districts of decision (Development, Aragon Network, ANT Traders) with a council on each of them. The Council of each district is always permeable to proposals but the decisions are taken by the Council. The Council can decide by voting in direct democracy, by consensus, for some decisions is delegated/liquid democracy (not a fan, because it's breaks the rule, one person one vote), for some decisions is restricted to x members that contributed with y, for some is strictly restricted to team/founders, and even artificial intelligence will have to be weighted to assess whether it looks for "the general welfare" as DAO democracy proposed.

Regardless of the structure, I think the start point is Who has the final call. At Aragon it's not clear for me, whether is the Aragon Network (that's my particular view), the ANT traders/holders, or the founders. Once that is clarified, the construction of Aragon's constitution will be easier.

**Comment by Smokyish** [comment]
No further changes will be made to this version, closing the issue.

**Comment by izqui** [comment]
We should clone the EIP format and merge a PR with a markdown file when a
proposal has been completed and acceoted
